### PR TITLE
Uncutfem 2D implementation

### PR DESCRIPTION
**Pull request by cekees**
This includes the 2D cutfem embedded method and several changes to to default parameter, tests, and equivalent polynomials implementation. I disabled the old ibm tests for now and will do a new PR with the 3D implementation and new cutfem tests for fsi applications.  Trying to make the PR's more manageable, but this one is already out of hand. /cc @jhcollins @zhang-alvin

### CODE REVIEW

File: proteus/mprans/RANS2P.py [hunk 1]
```diff
@@ -476,10 +534,10 @@ def attachModels(self, modelList):
                     self.ebq_n = None
                 self.ebqe_n = modelList[self.LS_model].ebqe[('grad(u)', 0)]
             else:
-                self.q_phi = 10.0 * numpy.ones(self.model.q[('u', 1)].shape, 'd')
-                self.phi_dof = -numpy.ones_like(self.model.u[0].dof)
-                self.ebqe_phi = 10.0 * numpy.ones(self.model.ebqe[('u', 1)].shape, 'd')
-                self.bc_ebqe_phi = 10.0 * numpy.ones(self.model.ebqe[('u', 1)].shape, 'd')
+                self.q_phi = -10.0 * numpy.ones(self.model.q[('u', 1)].shape, 'd')
```
Comment (zhang-alvin) [inline proteus/mprans/RANS2P.py:537]: What's the factor of 10 for in general? And what's the significance of the minus sign

File: proteus/tests/solver_tests_mprans/test_bochev_pressure_stabilization.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def test_bochev_pressure_cavity(load_cavity_problem,
 
     expected_path = 'comparison_files/' + 'comparison_' + 'twp_navier_stokes_cavity_2d' + '_p_t1.csv'
     #write comparison file
-    np.array(actual.root.p_t1).tofile(os.path.join(script_dir, expected_path),sep=",")
+    #np.array(actual.root.p_t1).tofile(os.path.join(script_dir, expected_path),sep=",")
```
Comment (zhang-alvin) [inline proteus/tests/solver_tests_mprans/test_bochev_pressure_stabilization.py:96]: good catch.

File: proteus/ADR.py [hunk 3]
```diff
@@ -274,28 +303,28 @@ def __init__(self,
         #cek come back
```
[the remark this answers is not in the document]
Comment (cekees) [reply proteus/ADR.py:303]: Yes, the ADR changes are  working towards the IIFEM implementation--next phase of development after the 3d cutfem.

File: proteus/mprans/RANS2P.py (hunk 1 above)
Comment (cekees) [reply proteus/mprans/RANS2P.py:537]: Those are just to give reasonable defaults that push the level set firmly into a single phase.

File: proteus/ADR.py [hunk 4]
```diff
@@ -17,6 +17,7 @@
 from proteus.TransportCoefficients import TC_base
 from proteus.SubgridError import SGE_base
 from proteus.ShockCapturing import  ShockCapturing_base
+from proteus.cADR import *
```
[the remark this answers is not in the document]
Comment (cekees) [reply proteus/ADR.py:20]: Agreed. Will get that cleaned up.

**Pull request merged**